.yamato/config/postprocessing.metafile

```diff
@@ -110,12 +110,12 @@ jobs:
         color_space: Linear
         test_platforms:
           - Standalone
-      - platform: Linux
-        api: Vulkan
-        build_config: mono
-        color_space: Linear
-        test_platforms:
-          - editmode
+      # - platform: Linux
```

Comment:
same as supra - why are we commenting and not plain deleting? Is it a temporary modification?